src/lib/arch/aarch64/Instruction_execute.cc

```diff
@@ -548,6 +548,7 @@ void Instruction::execute() {
         break;
       }
       case Opcode::AArch64_CASALW: {  // casal ws, wt, [xn|sp]
+        // TODO: Load and Store must occur atomically
```

Comment:
Should this TODO have been removed given we are not sure if it will be atomic in SST